api/accounts.go

```diff
@@ -407,3 +425,30 @@ func (a *API) tokenTransfersHandler(msg *apirest.APIdata, ctx *httprouter.HTTPCo
 	}
 	return ctx.Send(data, apirest.HTTPstatusOK)
 }
+
+// setSikHandler
+//
+//	@Summary		Updates the Secret Identity Key of an Address
+//	@Description	Updates the Secret Identity Key of an Address
+//	@Success		200	{object}	SikSet
+//	@Router			/accounts/sik [post]
+func (a *API) setSikHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
```

Comment:
I'm not sure of the need of this API method. It does nothing different from `/chain/transaction`.

We do have specific endpoints for some transactions that require special logic. Such as new election, which requires the Gateway to publish the metadata into IPFS. But for the rest of transactions, we use the generic endpoint which just relays the transaction.

Reply:
Perfect, it makes sense